Keymap Update

pcdunham opened the pull request:
Hello,

This is my first time attempting to contribute something on GitHub, so I hope I'm going through to right process.

I updated the ESC and NUM Lock keycodes with the QK prefix because the compiler was giving me errors. 

Updated the types for layer_state_set_user() to layer_state_t and  rgb_matrix_indicators_advanced_user() to bool and return false.

It compiles without errors and I flashed it to my board to test it.

pcdunham committed:
Updated ESC, NUM Lock keycodes, function types

Updated ESC, NUM Lock keycodes. Updated function types for layer_state_set_user and rgb_matrix_indicators_advanced_user so it would compile correctly.

pcdunham committed:
Updated ESC, NUM Lock keycodes, function types

Updated ESC, NUM Lock keycodes. Updated function types for layer_state_set_user and rgb_matrix_indicators_advanced_user so it would compile correctly.

pcdunham committed:
Merge pull request #1 from pcdunham/pcd_dev

Updated ESC, NUM Lock keycodes. Updated function types

skitzo2000 reviewed (approved):
Yep, you did it right!  Thank you for being a contributor!!!

Changes look sane.  The QK_GESC looks like something new! So thanks for keeping the repo up to date.  We should be able to get rid of that return false though, but it works for now.  I need to review in more depth, and probably make a few more adjustments.

pcdunham commented:
Cool. I’m only familiar enough with C to piece code snippets together and make minor edits. 

 

I wasn’t able to get yours to compile correctly until I changed the type of rgb_matrix_indicators_advanced_user() to bool and added return false. Someone from the QMK discord suggested that. I included a screenshot of the error. Hopefully you can figure that out. I have no clue.  Going forward, I’m going to put my changes here so I can change up the rgb settings to my preferences. https://github.com/pcdunham/qmk_firmware_fork/tree/pcd_dev/keyboards/gmmk/gmmk2/p96/ansi/keymaps/pcd

 

Cheers! 

 

Philip

 

 

 

 



From: Paul Norton ***@***.***> 
Sent: Tuesday, January 24, 2023 1:56 PM
To: skitzo2000/GMMK2-96 ***@***.***>
Cc: pcdunham ***@***.***>; Author ***@***.***>
Subject: Re: [skitzo2000/GMMK2-96] Keymap Update (PR #4)

 

@skitzo2000 approved this pull request.

Yep, you did it right! Thank you for being a contributor!!!

Changes look sane. The QK_GESC looks like something new! So thanks for keeping the repo up to date. We should be able to get rid of that return false though, but it works for now. I need to review in more depth, and probably make a few more adjustments.

—
Reply to this email directly, view it on GitHub <https://github.com/skitzo2000/GMMK2-96/pull/4#pullrequestreview-1268198855> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/A4YXTEM37ZDC74DNUZAKWQTWUAXVNANCNFSM6AAAAAAUFKYPSA> .
You are receiving this because you authored the thread.  <https://github.com/notifications/beacon/A4YXTEMYVZNLNITNML5CZ63WUAXVNA5CNFSM6AAAAAAUFKYPSCWGG33NNVSW45C7OR4XAZNRKB2WY3CSMVYXKZLTORJGK5TJMV32UY3PNVWWK3TUL5UWJTSLS4W4O.gif> Message ID: ***@***.*** ***@***.***> >
